### ISSUE TITLE
Error: UnimplementedError for compressWithList

### ISSUE THREAD

**Issue by Ballistx** [fluttercandies/flutter_image_compress#250]
### Content

I am using the following method to load an image which I want to draw on a canvas. However I always get the error message "Error: UnimplementedError" for compressWithList which exists twice. The one with the throw UnimplementedError() in class UnsupportedFlutterImageCompress and the actual one without throw UnimplementedError() in FlutterImageCompressPlatform.

`  Future<UI.Image> loadUiImage(
      String imageAssetPath, int height, int width) async {
    final ByteData data = await rootBundle.load(imageAssetPath);
    Uint8List uint8list = data.buffer.asUint8List();

    final Uint8List compressedData = await FlutterImageCompress.compressWithList(
      uint8list,
      minHeight: height,
      minWidth: width,
      quality: 100, // Adjust the quality level as needed
      //format: CompressFormat.png,
    );
    final codec = await UI.instantiateImageCodec(
      compressedData,
      targetHeight: height,
      targetWidth: width,
    );

    return (await codec.getNextFrame()).image;
  }`

**Comment by MzeeAlii** [comment]
Yep, same error.

**Comment by minhson3756** [comment]
Maybe you used it in another isolate, just use it in the main isolate

**Comment by lakshmanprabhu49** [comment]
@minhson3756 You are right, it works on main isolate. But this is pretty intensive process right? Do you know any technique to put it in a different isolate?

**Comment by GhagSagar23** [comment]
Any idea when will this functionality be implemented? @minhson3756.

**Comment by gcostaapps** [comment]
Any updates? As this task is CPU intensive it would be good if it could run outside the main isolate.
